Lacework-Integration-ContainerRegistry/lacework-integration-containerregistry.json

```diff
@@ -0,0 +1,107 @@
+{
+    "typeName": "Lacework::Integration::ContainerRegistry",
+    "description": "An example resource schema demonstrating some basic constructs and validation rules.",
+    "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git",
```

Comment:
We may want to change the source url to be this repos url and improve the `description` above

Reply:
@joshuadeanhall Done